### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by alexearnshaw**
This PR adds a pull request template with various checklists to help contributors and maintainers.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Thank you for your contribution to the Axway-Open-Docs repo.
+
+## Describe the changes
+
+Enter a brief description of your changes here to communicate to the maintainers why we should accept this pull request.
```
Comment (andreamussap) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: 'to communicate **why we should accept this pull request**' - This sounds a bit bossy, or threatening to me. Like 'we'll judge this and decide if we should accept it or not. I know that it's what we do but we could sound more amicable like what we say in the workshop 'Enter a brief description of your changes to communicate what you did and why'. Sounds more amicable IMHO :)

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Checklist for contributors
+
+Before submitting this PR, please make sure:
+
+* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
+* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
+* [ ] You have verified the technical accuracy of your change
+* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
+
+_Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._
```
Comment (andreamussap) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: 'Put an x in the boxes that apply' - This is creating more work for the contributor IMHO. It's ok to put a list with check boxes but I wouldn't ask them to actually check the boxes. Should be up to them. The most important is that they read/be aware of the list, so I'd start with 'If you're unsure about any of them,.....'

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Checklist for approvers
+
+_This section is to be filled out by project maintainers only._
+
+Before approving this PR, please make sure it:
+
+* [ ] Is clear and complete
```
Comment (andreamussap) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: 'Checklist for approvers' - could add to this list 'Check that there's no conflicts'

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (alexearnshaw) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I get your point on this one, but it's more a tip for how to check the boxes, as it wasn't obvious to me how to do it. Can't think of a better way to give the info without making it sound like they must do it, but I've removed the bit "you can fill these out after..."

**Pull request merged**